tests/integration/relations/test_opensearch_provider.py

```diff
@@ -584,66 +530,8 @@ async def test_admin_permissions_secrets(ops_test: OpsTest):
         assert "Error:" in results[0], results
 
 
+@pytest.mark.group(1)
 @pytest.mark.abort_on_fail
-@pytest.mark.usefixtures("only_without_juju_secrets")
-async def test_normal_user_permissions(ops_test: OpsTest):
```

Comment:
is this test **exactly identical** to the following one?

Reply:
@Mehdi-Bendriss not sure I follow. This is only run when we do not have juju secrets. In our CIs, that is 100% of the cases